client.go

```diff
@@ -53,13 +57,13 @@ func (c *PingClient) Run(addr string) error {
 			return nil
 		}
 
-		pong, err := c.Ping(addr)
+		pong, err := c.Echo(context.Background(), c.Next())
 		if err != nil {
-			return err
+			return fmt.Errorf("failed echo RPC call: %s", err)
```

Comment:
Much better error message, thanks!

Reply:
I just moved that from somewhere else.. Can't really take credit for that :wink: